README.md

```diff
@@ -101,6 +101,12 @@ On macOS, you can also install from the [Homebrew](https://brew.sh) repository:
 $ brew install vpn-slice
 ```
 
+You can also package it to RPM format, to keep track of installed files
```

Comment:
Should be "package it **in** RPM format."

It slightly annoys me that there isn't a `bdist_deb` to go along with this, but okay :joy:

Reply:
Thanks for correction there were also technical typos, 
actually there is `bdist_deb` provided by cryptically named package `stdeb` or `python3-stdeb`. Unfortunatelly this extension uses own configuration file.